### DISCUSSION TITLE
Is there a way to use this like dmenu / fzf / rofi?

### DISCUSSION THREAD

**Discussion by ykhan21** [Flow-Launcher/Flow.Launcher#2719] [Q&A]
Is there a way to use this like dmenu / fzf / rofi?
For example, can I run `ls ~/Pictures | flow.launcher` in a terminal, select a result, and get the selected result back in the terminal?

**Comment by Garulf** [comment]
Sorry at this time no info can be piped into Flow Launcher from the command line. 

But Flow Launcher's API uses fairly simple JSON that shouldn't be too hard to implement a quick script. 

Documentation can be found here:

https://www.flowlauncher.com/docs

**Comment by ykhan21** [comment]
@Garulf, are you talking about plugins? I will need to launch flow then trigger the plugin, right?

Or can I call the JSON RPCs from the command line?
________________________________

**Reply by Garulf** [reply]
Yes, you must make a plugin.
